main/tests/cypress/cypress/integration/project/grid/viewpanel-header/sort.spec.js

```diff
@@ -108,7 +105,6 @@ describe(__filename, function () {
     cy.get('.viewpanel-sorting a').contains('Sort').click();
     cy.get('.menu-container').contains('Reorder rows permanently').click();
     cy.assertNotificationContainingText('Reorder rows');
-    cy.reload();
```

Comment:
Interesting that this introduces flakiness! Intuitively those instructions are really part of the test: their purpose is to check that the reordering did indeed happen permanently (unlike the temporary sorting that is initially created). Perhaps there is a better way to assert this?

Reply:
I added back `cy.reload()` and fixed the flakiness with `cy.waitForProjectTable()` since the reload is similar to `cy.loadAndVisitProject()`.